data/tlg0732/tlg011/tlg0732.tlg011.1st1K-grc1.xml

```diff
@@ -0,0 +1,4487 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title>De anima libri mantissa</title>
+                <author>Alexander of Aphrodisias</author>
+                <editor>Ivo Bruns</editor>
+                <sponsor>HARVARD COLLEGE LIBRARY</sponsor>
```

Comment:
Make this title case instead of all caps.

Reply:
I'm not sure what you mean here. The title isn't all caps. Or should I change the sponsor?

@sonofmun

Reply:
Right, it is the sponsor that needs to be changed.